sql/2025/sustainability/ecommerce_bytes_per_type.sql

```diff
@@ -0,0 +1,167 @@
+#standardSQL
+# Median resource weights by ecommerce platform with detailed CO2e breakdown
+# Source: https://sustainablewebdesign.org/calculating-digital-emissions/
+# Declare variables to calculate the carbon emissions per gigabyte (kWh/GB)
+
+DECLARE grid_intensity NUMERIC DEFAULT 494;
+DECLARE embodied_emissions_data_centers NUMERIC DEFAULT 0.012;
+DECLARE embodied_emissions_network NUMERIC DEFAULT 0.013;
+DECLARE embodied_emissions_user_devices NUMERIC DEFAULT 0.081;
+DECLARE operational_emissions_data_centers NUMERIC DEFAULT 0.055;
+DECLARE operational_emissions_network NUMERIC DEFAULT 0.059;
+DECLARE operational_emissions_user_devices NUMERIC DEFAULT 0.080;
+
+WITH ecommerce_data AS (
+  SELECT
+    client,
+    page,
+    tech.technology AS ecommerce,
+    CAST(JSON_VALUE(summary, '$.bytesTotal') AS INT64) / 1024 AS total_kb,
```

Comment:
```suggestion
    INT64(summary.bytesTotal) / 1024 AS total_kb,
```
please update all the similar JSON values to this approach for readability